browser-sign-in/OktaBrowserSignIn/WelcomeViewController.swift

```diff
@@ -35,13 +38,51 @@ final class WelcomeViewController: UIViewController {
         }
     }
     
-    func navigateToDetailsPage() {
-        guard Credential.default != nil else { return }
-        performSegue(withIdentifier: "show-details", sender: self)
+    func SignInWithBiometrics() {
+        var error: NSError?
+        if context.canEvaluatePolicy(.deviceOwnerAuthentication, error: &error) {
+            context.evaluatePolicy(
+                .deviceOwnerAuthentication,
+                localizedReason: "Log in to your account",
+                reply: { success, error in
+                    guard let error = error else {
+                        DispatchQueue.main.async {
+                            self.storeSignInBehindBiometric()
+                        }
+                        return
+                    }
+                    self.show(title: "Error", error: error.localizedDescription)
+                })
+        } else {
+            show(title: "Error", error: error?.localizedDescription)
+        }
     }
     
-    @IBAction private func signInTapped() {
-        auth?.ephemeralSession = ephemeralSwitch.isOn
+    func storeSignInBehindBiometric() {
```

Comment:
Can you add headerdoc comments on this function, as well as the `signIn` function, to describe why one approach or the other is preferred?

I'd also suggest moving this function _below_ the regular `signIn` function.  Developers will most likely read this file from top-to-bottom, so the simplest function should be seen first.

Reply:
Good callout Mike!. I've updated this as well.